src/ec/curve25519/ops.rs

```diff
@@ -121,6 +121,72 @@ fn encode_point(x: Elem, y: Elem, z: Elem) -> EncodedPoint {
     bytes
 }
 
+#[cfg(all(target_arch = "x86_64", not(windows)))]
+pub fn scalar_mult_with_base_point(scalar: &Scalar) -> EncodedPoint {
+    let mut point = [0u8; ELEM_LEN];
+
+    unsafe { GFp_x25519_scalar_mult(&mut point, scalar, &BASE_POINT); }
+
+    point
```

Comment:
I ended up adding a comment about what we're calling, so I added the newlines back in.